src/Tribe/Customizer/Global_Elements.php

```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// overrides for common components/full/buttons/_border.pcss.
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt),
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt) {
+					border-color: <%= global_elements.accent_color %>;
+					color: <%= global_elements.accent_color %>;
```

Comment:
:warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 27801bc49daee7a89f2249e0cccd677c</sub>

Reply:
@paulmskim is this something we should account for in our linters?